libproc/lustre.c

```diff
@@ -131,6 +131,42 @@ typedef enum {
  * On EOF, functions return -1 with errno clear.
  */
 
+/*
+ * Fill-in the supplied version components with the Lustre version
+ * found in /sys.
+ *
+ * Returns -1 or less on error, 0 or greater on success.
+ */
+int
+sys_fs_lustre_version (pctx_t ctx, int *major, int *minor, int *patch,
+                       int *fix)
+{
+    int ret = -1;
+    char version[256];
+
+    if ((ret = proc_openf (ctx, PROC_FS_LUSTRE_VERSION)) < 0)
```

Comment:
Would it make sense to use `proc_scanf()` to simplify `sys_fs_lustre_version()` (at the cost of two open()/read()/closes())?